src/ProfessionalGraveyard.tsx

```diff
@@ -7,42 +7,82 @@ import { shortenAddress } from './Utilities';
 
 import Burned from './img/statistics-icons/Burned.png';
 
+import Filter from './Filter';
```

Comment:
If you applied the change above then filter should be exported as well as a named export so we can combine these two imports:

```jsx
import { Filter, Traits } from './Filter';
```